### PR TITLE
[Finishes #187355091] Setup API documentation using swagger

### PR DESCRIPTION
**Pull request by Celestin4**
**1. What does this PR do?**

Setup API documentation using swagger

**2. Description of task to be completed**

The task involves establishing API documentation utilizing Swagger, a popular framework for documenting APIs. This process will streamline understanding and usage of the API by providing clear and structured documentation. Through Swagger, developers can easily explore endpoints, parameters, payloads, and response formats. It ensures consistency and clarity in communication between developers and consumers of the API. Implementing Swagger enhances the overall development process and promotes better collaboration within the development team.

Relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187355091

**3. How should this be manually tested?**

- Try to access swagger UI by running the server and user this URL for swagger docs:
                         http://localhost:7000/docs/
-Try to write endpoints and make its documentation

**5. Screenshoots**
image

image

### CODE REVIEW

**Review by igorihimbazwe — APPROVED** [review]
This is good work

**Comment by pacifiquemboni** [comment]
this is good work but on my end api is not ending excution, it is continousily loading, double check and  let me know if it is on my end only

**Review by pacifiquemboni — APPROVED** [review]
cool👍

File: .DS_Store
Comment (UmuhireJessie) [inline .DS_Store:1]: Remove this file in stagged files

File: src/.DS_Store
Comment (UmuhireJessie) [inline src/.DS_Store:1]: Remove this file

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Well done @Celestin4, please do the following:
1. Check the comments on specific files
2. Change the commit message, i.e., it is Finishes not Finieshes

**Review by lilika67 — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**